yt/frontends/gizmo/data_structures.py

```diff
@@ -49,3 +54,113 @@ def _is_valid(cls, filename, *args, **kwargs):
 
     def _set_code_unit_attributes(self):
         super()._set_code_unit_attributes()
+
+    def _parse_parameter_file(self):
+        hvals = self._get_hvals()
+
+        self.dimensionality = 3
+        self.refine_by = 2
+        self.parameters["HydroMethod"] = "sph"
+        # Set standard values
+
+        # We may have an overridden bounding box.
+        if self.domain_left_edge is None and hvals["BoxSize"] != 0:
+            self.domain_left_edge = np.zeros(3, "float64")
+            self.domain_right_edge = np.ones(3, "float64") * hvals["BoxSize"]
+
+        self.domain_dimensions = np.ones(3, "int32")
+        self._periodicity = (True, True, True)
+
+        self.cosmological_simulation = 1
+
+        try:
+            self.current_redshift = hvals["Redshift"]
+        except KeyError:
+            # Probably not a cosmological dataset, we should just set
+            # z = 0 and let the user know
+            self.current_redshift = 0.0
+            only_on_root(mylog.info, "Redshift is not set in Header. Assuming z=0.")
+
+        try:
+            # The current version of GIZMO has updated the names of the Omegas
```

Comment:
Makes sense, I'll see if I can find exactly when things changed. I'm not sure there are any actual version numbers though, so it might just be a date/commit hash.

Reply:
Better than nothing. A date could also work.